File: infrastruktur/src/test/java/com/github/haschi/haushaltsbuch/RunCukesSystemIT.java
package com.github.haschi.haushaltsbuch;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(
        strict = true,
        tags = {"@system"}
)
public class RunCukesSystemIT
{
}

File: testit/src/test/kotlin/com/github/haschi/haushaltsbuch/RunCukesSystemIT.kt
package com.github.haschi.haushaltsbuch

import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber
import org.junit.runner.RunWith

@RunWith(Cucumber::class)
@CucumberOptions(strict = false, tags = ["@system"], plugin = ["pretty"])
class RunCukesSystemIT
